### DISCUSSION TITLE
Welcome to django-todo discussions!

### DISCUSSION THREAD

**Discussion by shacker** [shacker/django-todo#115] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vlad-Korolev** [comment]
Hello, This is a small and great app! Thanks to the author! 
Has anyone ever made a calendar with task rendering by day (bootstrap)?

**Reply by shacker** [reply]
Hi, and good question. No, this is the first time I've seen this feature request. Feel free to submit a PR if you have an idea for how this might be implemented.
